tests/MIRI_ecfs/S4_MIRI.ecf

```diff
@@ -20,7 +20,8 @@ sub_continuum	True	# Set True to subtract the continuum from the spectra using a
 highpassWidth	10		# The integer width of the highpass filter when subtracting the continuum
 
 # Parameters for sigma clipping
-sigma_clip		True	# Whether or not sigma clipping should be performed on the 1D time series
+clip_unbinned   False   # Whether or not sigma clipping should be performed on the unbinned 1D time series
```

Comment:
Let's set this to True for at least one of the test cases (arbitrarily MIRI perhaps) so that the relevant part of the code still gets tested